src/main/java/mate/academy/Main.java

```diff
@@ -2,16 +2,26 @@
 
 import java.time.LocalDate;
 import java.time.LocalDateTime;
+import mate.academy.lib.Injector;
 import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
 import mate.academy.model.MovieSession;
+import mate.academy.model.ShoppingCart;
+import mate.academy.model.User;
 import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
 import mate.academy.service.MovieSessionService;
+import mate.academy.service.OrderService;
+import mate.academy.service.ShoppingCartService;
+import mate.academy.service.UserService;
 
 public class Main {
+    public static final String BASE_PACKAGE_NAME = "mate.academy";
+    private static final Injector injector = Injector.getInstance(BASE_PACKAGE_NAME);
+
     public static void main(String[] args) {
-        MovieService movieService = null;
+        MovieService movieService = (MovieService) injector
+                .getInstance(MovieService.class);
```

Comment:
Consider using a generic method in the `Injector` to avoid explicit type casting. This will prevent potential `ClassCastException` if the `Injector` does not return the expected type.